### PR TITLE
Introduce purge-job

### PR DESCRIPTION
**Pull request by fridex**
## Related Issues and Dependencies

Related: https://github.com/thoth-station/purge-job/

## Does this require new deployment ?

- [x] Deployment for Test and Stage `AICoE/aicoe-cd` and Prod `operate-first/argocd-apps`.

## Description

Purging old data out of a Thoth deployment.

### CODE REVIEW

**Comment by fridex** [comment]
@harshad16 PTAL, tested in test environment, looks operational 🤞🏻 

/assign @harshad16

**Review by harshad16 — APPROVED** [review]
lgtm,
thanks :100:

File: purge-job/base/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+commonLabels:
+  app.kubernetes.io/name: thoth
+  app.kubernetes.io/component: purge-job
+  app.kubernetes.io/managed-by: aicoe-thoth-devops
+generatorOptions:
+  disableNameSuffixHash: true
+resources:
+  - imagestream.yaml
+  - argo-workflows/purge-template.yaml
+  - openshift-templates/purge-template.yaml
```
Comment (harshad16) [inline purge-job/base/kustomization.yaml:12]: oh the name different.
```suggestion
  - openshift-templates/purge.yaml
```

**Review by harshad16 — CHANGES_REQUESTED** [review]

File: purge-job/base/kustomization.yaml (hunk 1 above)
Comment (fridex) [reply purge-job/base/kustomization.yaml]: Thanks, adjusted 👍🏻

**Review by harshad16 — APPROVED** [review]
thanks :100:

**Pull request merged**